containerd/containerd.go

```diff
@@ -66,10 +66,18 @@ func (d *Driver) createContainer(image containerd.Image, containerName, containe
 		opts = append(opts, oci.WithPrivileged)
 	}
 
-	// Enable default seccomp profile.
+	if !config.Seccomp && config.SeccompProfile != "" {
+		return nil, fmt.Errorf("seccomp must be set to true, if using a custom seccomp_profile.")
+	}
+
+	// Enable default (or custom) seccomp profile.
 	// Allowed syscalls for the default seccomp profile: https://github.com/containerd/containerd/blob/master/contrib/seccomp/seccomp_default.go#L51-L390
 	if config.Seccomp {
-		opts = append(opts, seccomp.WithDefaultProfile())
+		if config.SeccompProfile != "" {
+			opts = append(opts, seccomp.WithProfile(config.SeccompProfile))
+		} else {
+			opts = append(opts, seccomp.WithDefaultProfile())
```

Comment:
Maybe this is your goal of the bool and the profile :D. Provide defaults.

Reply:
Yeah, this is by design 🙂 . Elaborated [`here`](https://github.com/Roblox/nomad-driver-containerd/pull/38#issuecomment-685076881)